README.markdown

```diff
@@ -15,6 +15,16 @@ Place this script into the **.git/hooks/** directory in your repository. It must
 
     book]$ ln -s ~/git-spell-check/pre-commit .git/hooks/pre-commit
 
+You will need to install [`aspell`](http://aspell.net/).
```

Comment:
Please use HTTPS links.

Reply:
Aspell doesn't have https support, unfortunately.

Reply:
Ah, that's too bad. I guess let's keep it as http://aspell.net.